governance/meeting-host-playbook.md

```diff
@@ -0,0 +1,85 @@
+# TAG Environmental Sustainability Meeting Host Playbook
+
+Meetings are held to coordinate and discuss TAG ENV activities. The TAG ENV leads host these meetings. The following playbook can be used for reference to structure the meeting, you don't need to stick step by step (word by word), it should give guidance how a meeting usually is structured.
+
+## Requirements to host the meeting
+
+* Install Zoom and check your settings (especially audio and video).
+* Get access to the [CNCF TAG YouTube Channel](https://www.youtube.com/@CNCFEnvTAG) to start the livestream.
+
+## Pre-Meeting
+* Add a new blank agenda entry to the meeting notes by copying the template and editing the details accordingly.
+* Announce a day in advance in Slack to add agenda items to meeting notes.
+* Announce the meeting about an hour before in the `#tag-env-sustainability` Slack channel.
+
+```
+Hi folks! The CNCF TAG ENV bi-weekly meeting takes place 1 hour from now
+Links:
+* Agenda & Minutes: https://bit.ly/cncf-tag-env-meeting-notes
+* Zoom Link: https://zoom.us/my/cncftagenvsustainability, passcode `77777`
+
+If anyone has anything they’d like to discuss, please add to it to the agenda, thanks!
+```
+
+## During the Meeting
+
+### Preparation
+
+* Slack reminder (thread) *"The meeting start now!"*
+* [Join](https://zoom.us/my/cncftagenvsustainability) the meeting
+* Turn on your camera
+* Say hello to everyone 
+  * *"Hello all, let's wait a few minutes for everyone to join"*
+* Post the meeting notes to the meeting chat
+  * *"I posted the meeting notes link to chat, please add yourself to the attendees list, thank you!"*
+* Prepare the YouTube livestream
+* Open meeting notes and share screen
+* Wait until 3 minutes have passed. Talk to people during this time so that the session does not get stalled. If many people are missing, you may want to wait a minute longer.
```

Comment:
I usually do 5, because at least here everyone is 5 minutes late.

Reply:
👍